### DISCUSSION TITLE
Generated PDFs should be accessible and searchable

### DISCUSSION THREAD

**Discussion by Moini** [baimard/gestion#186] [Q&A]
Currently, the generated PDF files consist of an embedded raster image with a fixed width of 1000px.

This is bad in multiple ways:

* the PDF files are not searchable
* the PDF files do not scale (they are blurry already at normal zoom)
* the PDF files lack metadata
* blind persons have no means to read the pdf with a screenreader

To fix this, please use a pdf generator that generates PDF files which:

* include text as text
* include at least some metadata
* only use vector shapes instead of raster graphics elements

**Comment by baimard** [comment]
This is not a bug, this need to be open in a new discussion : https://github.com/baimard/gestion/discussions

Also, to prevent a duplicate discussion, please take the time to read all the opens discussions.

**Comment by Moini** [comment]
To me, it is a bug / bad implementation choice. Is there a way to just move a bug report to a 'discussion' in GitHub?

**Comment by Moini** [comment]
See https://github.com/github/feedback/discussions/2952

**Comment by baimard** [comment]
> To me, it is a bug / bad implementation choice. Is there a way to just move a bug report to a 'discussion' in GitHub?

no it's not a bug bad implementation. it's a feature simply, if you wan't to try an other implementation :

- FORK
- dev
- test
- PR
